client/tailscale/localclient.go

```diff
@@ -859,6 +859,32 @@ func (lc *LocalClient) NetworkLockSign(ctx context.Context, nodeKey key.NodePubl
 	return nil
 }
 
+// SetServeConfig sends a ServeConfig establishing serve and ingress settings.
```

Comment:
// SetServeConfig sets or replaces the serving settings.
// If config is nil, settings are cleared and serving is disabled.